spec.bs

```diff
@@ -2939,31 +2940,42 @@ might choose to require a [=k-anonymity=] threshold of fifty users over a seven
 will maintain the count over the chosen duration and compare the count to the chosen [=k-anonymity=]
 threshold when responding to [=query k-anonymity count=].
 
+The [=user agent=] must maintain a <dfn>k-anonymity cache</dfn> as a [=map=]
+whose [=map/keys=] are [=SHA-256=] hashes of the k-anonymity key [=string=]s for all of the ads and ad components in

```

Comment:
link to [=interest group/ads=] and [=interest group/ad components=]

Reply:
done